Werewolf for Telegram/Werewolf Control/Handlers/UpdateHandler.cs

```diff
@@ -438,6 +438,31 @@ internal static void HandleUpdate(Update update)
                                             id);
                                         return;
                                     }
+                                    // If command is not allow outside topic, or it is not admin / dev command. Check topic id and restrict to topic.
+                                    // Kmoh COOKED : (Oops... i forgot another thing that i had to write down. It was in my mind a second ago...)
+                                    if (!(command.AllowOutsideConfiguredTopic || command.GlobalAdminOnly || command.LangAdminOnly || command.GroupAdminOnly || command.DevOnly) )
```

Comment:
I think group admin commands should not be possible across all topics per se. E.g. `/smite` or `/forcestart` only make sense in the topic where the game happens. 

Thinking about it, I'd say *all* commands for ordinary users and group admins should only work in the selected topic, except for the `/settopic` / `/remtopic` commands. So what that means is:
- Remove `command.GroupAdminOnly` in this line
- Do not mark any commands except for the topic selection commands as `AllowOutsideConfiguredTopic`

Global admin / lang admin / dev commands can simply work across all topics like you've written it, that's fine.